STYLE_GUIDE.md

```diff
@@ -0,0 +1,88 @@
+# Style Guide
+
+This document outlines the style guide for the `epinowcast` package. This guide is a work in progress and will be updated as the package evolves. We welcome contributions to this guide and encourage you to raise issues or submit PRs if you have any suggestions.
+
+In addition to this guide we also follow the [tidyverse style guide](https://style.tidyverse.org/). This guide is a subset of the `tidyverse` style guide and outlines the additional style requirements for the `epinowcast` package.
+
+## Naming conventions
+
+- We use a `enw_` prefix to delineate functions that are part of the `epinowcast` package. This is to avoid conflicts with other packages and to make it clear to users which functions are part of the package.
+- The use of this prefix is not required for internal functions (i.e. functions that are not exported) or that are unlikely to have naming conflicts with other packages.
+
+## Dependencies
+
+In general we aim to minimise dependencies on other packages where possible. This makes it easier to maintain the package and reduces the risk of breaking changes in other packages impacting our users. However, additional dependencies are sometimes necessary to improve the functionality of the package.
+
+The following guidelines should be followed when using adding dependencies:
+
+- Added to the `Imports` or `Suggests` field `DESCRIPTION` file in alphabetical order.
```

Comment:
provide guidance about when imports vs suggests? my rough take:
 - imports: must be present to use the package
 - suggests: may be required to building / test / etc package; also, may provide some extended capability

Reply:
also, to what extent does "importFrom" cause imports to be updated automatically?

Reply:
Import from doesn't cause imports to be added to the DESCRIPTION but it does manage the namespace for you. 

Agree on the usage split for imports and suggests. Also worth saying the bar for a suggests is lower than for an import. We don't need to get to verbose here though as for most people who get this far this is likely to be common knowledge.